app/javascript/components/patient/assessment/actions/CurrentStatus.js

```diff
@@ -132,19 +132,32 @@ class CurrentStatus extends React.Component {
     }
   }
 
+  getHeaderLabel = () => {
+    let eligibilityStatus = this.props.report_eligibility.household
+      ? 'In a household'
+      : this.props.report_eligibility.eligible
+      ? 'Eligible for notifications'
+      : this.props.report_eligibility.reported
+      ? 'Received a report today'
+      : this.props.report_eligibility.sent
+      ? 'Waiting for a response'
+      : 'Not eligible for notifications';
+    return ` Notification Status is: ${eligibilityStatus}.`;
+  };
+
   render() {
     return (
       <React.Fragment>
-        <h4 className="pb-3">
+        <h2 className="tertiary-title pb-3">
           <b>
             {this.props.isolation ? 'Isolation' : 'Exposure'} Workflow: {this.generateStatus(this.props.status)}
           </b>
           {this.generateStatusTooltip(this.props.status)}
           <span className="b-border-right-3 pl-3"></span>
-          <b className="pl-3">
+          <b className="pl-3" aria-label={this.getHeaderLabel()}>
```

Comment:
The screen-reader doesnt lock on the eligibility tooltip, so put the aria-label here instead.